P5/Source/Specs/model.identInfo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<classSpec xmlns="http://www.tei-c.org/ns/1.0" module="tei" xml:id="IDENTINFO" type="model" ident="model.identInfo">
+  <desc versionDate="2022-09-09" xml:lang="en">groups elements which provide an explanation or an equivalency for a markup construct.</desc>
+  <desc versionDate="2007-12-20" xml:lang="ko">마크업 구성에 대한 대체명, 설명 또는 기술을 제시하는 요소를 모아 놓는다.</desc>
```

Comment:
@sydb  Delete lines 12-20 (translations from the old model.glossLike).

Reply:
Done.